src/corona_hakab_model/generation/matrix_consts.py

```diff
@@ -23,5 +23,7 @@ class MatrixConsts(NamedTuple):
     }
     use_parasimbolic_matrix = True,
     clustering_switching_point = 20,
+    community_member_edges = 2, #m
```

Comment:
the community member edges (m) is half the average amount of connections per node. 
in this case, the average amount of connections per node, is given by the daily + weekly amount of connections, for the specific connection type.
since m will be a float, when adding the nodes, some will add math.ceil(m) connections, and some math.floor(m) connections.
u can use the random_round() method, located at the bottom of matrix_generator.py, to choose who will be rounded up and who will be rounded down.

Reply:
fixed